### PR TITLE
Mesoshapes county 1.0 cleanup - S Korea, N Korea, Bhutan, Bolivia

### PR DESCRIPTION
**Pull request by stepps00**
This PR removes duplicate county features in South Korea, North Korea, Bhutan, and Bolivia.
Using two modified `add-field.py` scripts ([here](https://github.com/mapzen/whosonfirst-toolbox/commit/8081f034145eac72c09a331fe5cc4f0f85fa00a2)), duplicate features were matched, and superseding/deprecating work was done. 

If Who's On First already knew about this feature prior to the mesoshapes import, I retained that feature (and original properties) and deprecated the leftover feature. If the feature was new to Who's On First, one record was chosen at random and the leftover feature was deprecated.

This should be merge-able to `master` as-is; no additional PIP-ing is required.

Japan has some additional duplicate features, which will come in a separate PR.

### CODE REVIEW

File: data/110/874/684/3/1108746843.geojson [hunk 1]
```diff
@@ -43,13 +43,15 @@
...
-    "wof:lastmodified":1480457902,
+    "wof:lastmodified":1481070951,
     "wof:name":"Anak",
     "wof:parent_id":85676605,
     "wof:placetype":"county",
     "wof:repo":"whosonfirst-data",
     "wof:superseded_by":[],
-    "wof:supersedes":[],
+    "wof:supersedes":[
+        1108747711
```
Comment (nvkelso) [inline data/110/874/684/3/1108746843.geojson]: This record `1108747711` record other needs the `superseded_by` hooha.

File: data/110/874/684/5/1108746845.geojson [hunk 2]
```diff
@@ -43,13 +43,15 @@
...
-    "wof:lastmodified":1480457902,
+    "wof:lastmodified":1481070951,
     "wof:name":"Anbyon",
     "wof:parent_id":85676619,
     "wof:placetype":"county",
     "wof:repo":"whosonfirst-data",
     "wof:superseded_by":[],
-    "wof:supersedes":[],
+    "wof:supersedes":[
+        1108747713
```
Comment (nvkelso) [inline data/110/874/684/5/1108746845.geojson]: Ditto

File: data/110/874/684/7/1108746847.geojson [hunk 3]
```diff
@@ -43,13 +43,15 @@
...
-    "wof:lastmodified":1480457902,
+    "wof:lastmodified":1481070951,
     "wof:name":"Anju City",
     "wof:parent_id":85676637,
     "wof:placetype":"county",
     "wof:repo":"whosonfirst-data",
     "wof:superseded_by":[],
-    "wof:supersedes":[],
+    "wof:supersedes":[
+        1108747715
```
Comment (nvkelso) [inline data/110/874/684/7/1108746847.geojson]: Ditto

Comment (nvkelso) [inline data/110/874/684/7/1108746847.geojson]: This was hiding in another commit.

File: data/110/874/684/3/1108746843.geojson (hunk 1 above)
Comment (nvkelso) [inline data/110/874/684/3/1108746843.geojson]: This was hiding in another commit.

**Comment by nvkelso** [comment]
LGTM

**Comment by nvkelso** [comment]
Ready for re-pipping (today), or we just merge it by EOD Wednesday (after updating meta files) for upcoming blog post.

**Comment by nvkelso** [comment]
Bump.

**Comment by stepps00** [comment]
Ideally we would re-PIP features in South Korea, but since these `county` features are do not have any descendants (see: [this issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/547)), we should be OK with merging this branch to `master`.

**Comment by thisisaaronland** [comment]
I will try to get to these today.

**Comment by thisisaaronland** [comment]
Merged as of https://github.com/whosonfirst-data/whosonfirst-data/compare/0335f0d95102...a467db0b3045

**Pull request merged**